onnxruntime/core/providers/cuda/cudnn_common.h

```diff
@@ -60,6 +60,53 @@ class CudnnFilterDescriptor final {
   cudnnFilterDescriptor_t desc_;
 };
 
+class CudnnDropout final {
```

Comment:
As we discussed offline, CudnnDropout is unnecessary for inferencing. We can delete it if training does not need it.

Reply:
ORT will support training. :)